apiary.apib

```diff
@@ -363,6 +363,8 @@ Telematics Data Model* for object descriptions as well.
 + multidayBasis     :                                  0 (required, number) - Multiday basis (7 or 8) used by the motor carrier to compute cumulative duty hours
 + outputFileComment : `fake Duty Status Log for testing` (required, string) - A textual field that may be populated with information pertaining to the creation of an ELD output file
 
++ eventDataChecksum :                                    (required, string) - The hexidecimal value result of a bitwise exclusive OR(XOR) operation using Table 3 of the ELD mandate
```

Comment:
In APIblueprint if you put a colon after the element name then you need to have a sample value after the colon.

I think this element is fine without a sample value. So you could just delete the colon